### DISCUSSION TITLE
Correlation between Trace, Log and Metrics in Opentelemetry using .Net

### DISCUSSION THREAD

**Discussion by neerajCL** [open-telemetry/community#1675] [Community Discussions]
Is there any way to correlate between Trace, Log and Metrics in Opentelemetry using .Net/C#
I am able to correlate between Trace and log using OpenTelemetry in .net and able to send the data to Elastic APM Server. 
Same trace id present in Trace and Logs. Is there any way to Correlate between Trace, Log and Metrics in OpenTelemetry using .Net

**Comment by trask** [comment]
hi @neerajCL! typically correlation with metrics would be done using [exemplars](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#exemplar), but these are not marked stable yet in the OpenTelemetry specification, so I'd recommend asking in https://github.com/open-telemetry/opentelemetry-dotnet to see if there's any experimental support for them yet
